examples/Tutorial 5 - Graph Coloring Solver.ipynb

```diff
@@ -0,0 +1,207 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6096a1cc",
+   "metadata": {},
+   "source": [
+    "In this notebook, we are going to implement a graph coloring algorithm based on a MIS solver for a practical problem of sharing a bandwidth of frequencies for a network of antennas."
```

Comment:
Could you please add a title ?

Reply:
And a little more narrative on the algorithm and implementation ? You can have a look at Tutorial 1.